_posts/en/newsletters/2021-12-08-newsletter.md

```diff
@@ -0,0 +1,166 @@
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23155][] extends the `dumptxoutset` RPC with the hash
+  of the chainstate snapshot (UTXO set) along with the number of
+  transactions in the entire chain up until that point.  This
+  information can be published alongside the chainstate so that others
+  can verify it using the `gettxoutsetinfo` RPC, allowing it to be used
+  with the proposed [assumeUTXO][topic assumeutxo] node bootstrapping.
+
+- [Bitcoin Core #22513][] allows `walletprocesspsbt` to sign without
+  finalizing the [PSBT][topic psbt] workflow. This is useful for
+  complex scripts, for example, in a [tapscript][topic tapscript] with
+  two paths: a fallback script path that requires only signer Alice,
+  plus a normal path with multiple signers including Alice. When Alice
+  signs, it is best to delay finalizing the PSBT with the fallback
+  script path and instead construct a PSBT containing both of Alice’s
+  signatures, pass the PSBT to the other signers, and wait for
+  them to sign. In this scenario, the ultimate path is determined after
+  all signatures are produced.
+
+- [C-Lightning #4921][] updates the implementation of [onion
+  messages][topic onion messages] to match the latest updates to the
+  draft specifications for [route blinding][bolts #765] and [onion
+  messages][bolts #759].
+
+- [C-Lightning #4829][] adds experimental support for the proposed
+  LN protocol change in [BOLTs #911][] that will allow nodes to advertise
+  their DNS address rather than their IP address or Tor onion service
+  address.
+
+- [Eclair #2061][] adds initial support for [onion messages][bolts #759]. Users
+  can enable the `option_onion_messages` feature to relay onion messages and
+  send onion messages with the `sendonionmessage` RPC. Handling incoming onion
+  messages and [route blinding][bolts #765] are not yet implemented.
+
+- [Eclair #2073][] adds support for the optional channel type negotiation
+  feature bit as specified in draft [BOLTs #906][].  This corresponds
+  to LND's implementation of the same draft feature [last week][news177
+  lnd6026].
+
+- [Rust-Lightning #1163][] allows the remote party to set their channel
+  reserve below the dust limit, even all the way down to zero.  In the
+  worst case, this allows the local node to costlessly attempt to steal
+  funds from a fully-spent channel---although such a theft attempt will
+  still fail if the remote party is monitoring their channels.  By
+  default, most remote nodes discourage such attempts by setting a
+  reasonable channel reserve, but some Lightning Service Providers
+  (LSPs) use low or zero channel reserves order to provide users with a
+  better experience---allowing them to spend 100% of the funds in the
+  channel.  Since only the remote node is taking any risk, there's no
+  problem allowing the local node to accept such channels.
+
```

Comment:
I think https://github.com/lightning/bolts/pull/940 is worthwhile to be mentioned.

The support for TOR v2 Onions was being deprecated and Lightning Nodes should not announce TOR v2 Onion addresses in their `node_announcements` anymore. Rust Lightning has also implemented this in https://github.com/lightningdevkit/rust-lightning/pull/1204

Reply:
@lightning-developer thanks!  Per [our schedule](https://gist.github.com/harding/0dffdde00761262ca094219c33b8ff22) we'll be considering it for mention next week, since it was only merged two days ago.

Reply:
Ok I have just provided you with a patch at https://github.com/harding/bitcoinops.github.io/pull/22/files I did not know about your schedule. Sorry for the delay.